src/codeManager.ts

```diff
@@ -345,6 +345,8 @@ export class CodeManager implements vscode.Disposable {
 
         if (this._codeFile) {
             const codeFileDir = this.getCodeFileDir();
+            const config = vscode.workspace.getConfiguration();
+            const ppath = config.get('python.pythonPath');

```

Comment:
better naming: ppath -> pythonPath

Reply:
Please fix the CI error. I guess `config.get('python.pythonPath')` should be `config.get<string>('python.pythonPath')`